mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/Connection.cs

```diff
@@ -751,10 +754,12 @@ class PacketReader {
 
 				// For reply packets
 				offset = 0;
-				ReadInt (); // length
+				var len = ReadInt (); // length
 				ReadInt (); // id
 				ReadByte (); // flags
 				ErrorCode = ReadShort ();
+				if (ErrorCode == (int)Mono.Debugger.Soft.ErrorCode.INVALID_ARGUMENT && len > offset)
+					ErrorMsg = ReadString ();
```

Comment:
Is there a protocol bump for this?

Reply:
There was not in the PR upstream: https://github.com/mono/mono/pull/19248

I can ask there.

Reply:
https://github.com/mono/mono/pull/19248#discussion_r396620830

Looks like the answer is "No, but yes."